lib/locales/ko.yml

```diff
@@ -0,0 +1,22 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/lib/locales/utils/p11n.rb)
+
+ko:
+  pagy:
+
+    item_name:
+      one: "항목"
```

Comment:
I think `:one` is not used in the `ko` pluralization (which should be `:other` type)